RichClipboard.package/RichClipboard.class/instance/initialize.st

```diff
@@ -4,6 +4,7 @@ initialize
 	
 	openWindows := LinkedList new.
 	storage := RCStorage new.
+	removeFromClipboardTimeoutMS := 60000.
```

Comment:
Ich glaube eine eine default Variable auf RichClipboard (Klassenseite) wäre besser (auch Variable aus Klasse entfernen)

Und wie gesagt könnt ihr mit isTestInstance prüfen, ob ihr den default Timeout oder den verkürtzten test Timeout nutzen wollt

Reply:
Ah okay, vllt. habe ich dein Feedback von oben nicht so richtig verstanden. Um das noch zu klären, verstehe ich das jetzt wie folgt:

Wir haben ein default Variable auf Klassenseite sowie ein Getter auf Instanzseite, der prüft, ob wir gerade in einem TestInstance sind. Wenn ja, setzen wir das Timeout auf 10 ms oder etwas. Wenn nicht, nutzen wir das default Timeout.

Ist das richtig so? Wir haben auch nach wie vor keine Tests dafür.

Reply:
Achso wenn wir keine Tests haben ist das ganze natürlich nicht nötig.

Ich würde die Variable trotzdem zu einer Default Variable umbauen

Reply:
Alles klar, dann baue ich jetzt erstmal nur die Default Variable ein.